### ISSUE TITLE
steam doing john cena

### ISSUE THREAD

**Issue by CaptainDumb** [ValveSoftware/steam-for-linux#10532]
#### Your system information

* Steam client version (build number or date): 1:1.0.0.79 (from .deb version)
* Distribution (e.g. Ubuntu): Mint 21.3
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes
* Steam Logs: [generate by running this command in a terminal `tar -zcvf ~/Desktop/steam-logs.tar.gz ~/.steam/steam/logs`]
* GPU: advanced micro devices

#### Please describe your issue in as much detail as possible:
the steam client is completely fucking invisible
![image](https://github.com/ValveSoftware/steam-for-linux/assets/101341925/d8991977-9faa-43f2-9a8b-37d4785c928b)
btw the gui seen on there is from search i did to partially see the gui

#### Steps for reproducing this issue:

1.  launch steam 
2. profit (or not idk)

**Comment by Greeedling** [comment]
See if you can open the settings menu. Check under interface and see if 'Enable GPU accelerated rendering in web views' or 'Enable hardware video decoding' are active. Disabling those fixed the transparency issue for me.

**Comment by CaptainDumb** [comment]
the only thing that was responding was the friends menu

**Comment by lavadrop** [comment]
I just experienced this and I can trigger it. I'm using Kvantum with KDE Plasma 6.0.4.
If I use ANYTHING other than Breeze themes, color schemes or applications styles, Steam's UI is invisible except for the friends list.
This I get when running from terminal:

```
steam.sh[16663]: Running Steam on opensuse-tumbleweed 20240418 64-bit
steam.sh[16663]: STEAM_RUNTIME is enabled automatically
setup.sh[16736]: Steam runtime environment up-to-date!
steam.sh[16663]: Steam client's requirements are satisfied
[2024-04-21 17:54:55] Startup - updater built Apr 17 2024 01:04:18
[2024-04-21 17:54:55] Startup - Steam Client launched with: '/home/user/.local/share/Steam/ubuntu12_32/steam'
[2024-04-21 17:54:55] Opted in to client beta 'publicbeta' via beta file
You are in the 'publicbeta' client beta.
04/21 17:54:55 minidumps folder is set to /tmp/dumps
04/21 17:54:55 Init: Installing breakpad exception handler for appid(steam)/version(1713317598)/tid(16780)
[2024-04-21 17:54:55] Loading cached metrics from disk (/home/user/.local/share/Steam/package/steam_client_metrics.bin)
[2024-04-21 17:54:55] Using the following download hosts for Public, Realm steamglobal
[2024-04-21 17:54:55] 1. https://client-update.akamai.steamstatic.com, /, Realm 'steamglobal', weight was 1000, source = 'update_hosts_cached.vdf'
[2024-04-21 17:54:55] 2. https://cdn.cloudflare.steamstatic.com, /client/, Realm 'steamglobal', weight was 1, source = 'update_hosts_cached.vdf'
[2024-04-21 17:54:55] 3. https://cdn.steamstatic.com, /client/, Realm 'steamglobal', weight was 1, source = 'baked in'
[2024-04-21 17:54:55] Verifying installation...
[2024-04-21 17:54:55] Verification complete
UpdateUI: skip show logo
Steam logging initialized: directory: /home/user/.local/share/Steam/logs

XRRGetOutputInfo Workaround: initialized with override: 0 real: 0xea77df50
XRRGetCrtcInfo Workaround: initialized with override: 0 real: 0xea77c5f0
steamwebhelper.sh[16814]: === dom 21 abr 2024 17:54:55 CST ===
steamwebhelper.sh[16814]: Starting steamwebhelper under bootstrap sniper steam runtime at /home/user/.local/share/Steam/ubuntu12_64/steam-runtime-sniper
CAppInfoCacheReadFromDiskThread took 79 milliseconds to initialize
Steam Runtime Launch Service: starting steam-runtime-launcher-service
Steam Runtime Launch Service: steam-runtime-launcher-service is running pid 17008
bus_name=com.steampowered.PressureVessel.LaunchAlongsideSteam
/usr/share/themes/Breeze-Dark/gtk-2.0/widgets/entry:70: error: unexpected identifier 'direction', expected character '}'
/usr/share/themes/Breeze-Dark/gtk-2.0/widgets/styles:36: error: invalid string constant "combobox_entry", expected valid string constant
BRefreshApplicationsInLibrary 1: 1ms
BuildCompleteAppOverviewChange: 780 apps
RegisterForAppOverview 1: 26ms
RegisterForAppOverview 2: 26ms
[2024-04-21 17:55:32] Shutdown
```
console_log.txt
```
[2024-04-21 17:54:59] Client version: 1713317598
[2024-04-21 17:54:59] wayland: skipping init because GAMESCOPE_WAYLAND_DISPLAY is not set
[2024-04-21 17:54:59] Warning: wayland init failed, some functionality may be unavailable
[2024-04-21 17:54:59] Loaded SDL version 3.0.0-3930-gd79f86525
[2024-04-21 17:54:59] GetWin32Stats: display was not open yet, good
[2024-04-21 17:54:59] GetWin32Stats: display was not open yet, good
[2024-04-21 17:54:59] wayland: skipping init because GAMESCOPE_WAYLAND_DISPLAY is not set
[2024-04-21 17:54:59] Steam Runtime Launch Service: starting steam-runtime-launcher-service
[2024-04-21 17:54:59] Steam Runtime Launch Service: steam-runtime-launcher-service is running pid 17008
[2024-04-21 17:54:59] Failed to init SteamVR because it isn't installed
[2024-04-21 17:54:59] GLibLog: domain:Gtk  msg:gtk_disable_setlocale() must be called before gtk_init()
[2024-04-21 17:54:59] GLibLog: domain:Gtk  msg:Failed to load module "appmenu-gtk-module"
[2024-04-21 17:54:59] GLibLog: domain:Gtk  msg:Unable to locate theme engine in module_path: "adwaita",
[2024-04-21 17:54:59] local (potentially out of sync) copy of roaming config loaded - 1481 bytes.
[2024-04-21 17:54:59] roaming config store loaded successfully - 1481 bytes.
[2024-04-21 17:54:59] migrating temporary roaming config store
[2024-04-21 17:54:59] System startup time: 3.65 seconds
[2024-04-21 17:54:59] ExecCommandLine: "'/home/user/.local/share/Steam/ubuntu12_32/steam'"
```

**Comment by err0r2k** [comment]
Same problem here but with Linux Mint 21.3 and Cinnamon.

What worked:
Starting Steam with `steam -cef-disable-gpu ` and  Settings -> Interface -> Disble Gpu Acceleration in Web View.

Big Picture Mode also worked for me:
Settings -> Disable GPU Blocklist

**Comment by LiamTheNinja5** [comment]
I am only getting this issue after I have updated to the steam family beta

**Comment by LiamTheNinja5** [comment]
> Same problem here but with Linux Mint 21.3 and Cinnamon.
> 
> What worked: Starting Steam with `steam -cef-disable-gpu ` and Settings -> Interface -> Disble Gpu Acceleration in Web View.
> 
> Big Picture Mode also worked for me: Settings -> Disable GPU Blocklist

Disabling Gpu Acceleration in Web View before switching to family beta fixed my issue
Thank you

**Comment by MarcosCC** [comment]
I am also having this issue (in Debian testing/trixie)

This does work: Settings -> Interface -> Disable Gpu Acceleration in Web View

But weirdly it also works if I start steam from the console (just typing "steam")

This started happening about a month ago.

**Comment by vrement** [comment]
Same thing: Steam is completely invisible! Tried: 

rm -rfv ~/.steam ~/.local/share/Steam ~/.steampath ~/.steampid

and also tried: steam -cef-disable-gpu

Also removing and reinstalling did NOTHING. On Ubuntu Steam has always worked no matter if a deb or a snap package. Now that i thought to give Mint a shot for a long time on one machine to just enjoy gaming it's nothing but trouble and tinkering? Hard to believe that Mint is based on Ubuntu as same things often are just messed up on Mint. I wonder why is this. 

Description:	Linux Mint 21.3 XFCE
Release:	21.3
Codename:	virginia

For the record when the command steam -cef-disable-gpu is issued all kind of stuff appears happening in the terminal for the time being and then everything stops and Steam does no even appear to taskbar. Nevermind opening as visible.

`steam -cef-disable-gpu
steam.sh[2205]: Running Steam on linuxmint 21.3 64-bit
steam.sh[2205]: STEAM_RUNTIME is enabled automatically
setup.sh[2275]: Steam runtime environment up-to-date!
steam.sh[2205]: Steam client's requirements are satisfied
[2024-06-22 13:32:04] Startup - updater built Jun 20 2024 17:13:49
[2024-06-22 13:32:04] Startup - Steam Client launched with: '/home/tux/.steam/debian-installation/ubuntu12_32/steam' '-srt-logger-opened' '-nominidumps' '-nobreakpad' '-cef-disable-gpu'
06/22 13:32:04 minidumps folder is set to /tmp/dumps
06/22 13:32:04 Init: Installing breakpad exception handler for appid(steam)/version(1718904662)/tid(2344)
[2024-06-22 13:32:05] Loading cached metrics from disk (/home/tux/.steam/debian-installation/package/steam_client_metrics.bin)
[2024-06-22 13:32:05] Failed to load cached hosts file (File 'update_hosts_cached.vdf' not found), using defaults
[2024-06-22 13:32:05] Using the following download hosts for Public, Realm steamglobal
[2024-06-22 13:32:05] 1. https://cdn.steamstatic.com, /client/, Realm 'steamglobal', weight was 1, source = 'baked in'
[2024-06-22 13:32:05] Verifying installation...
[2024-06-22 13:32:05] Verification complete
UpdateUI: skip show logo
Steam logging initialized: directory: /home/tux/.steam/debian-installation/logs

XRRGetOutputInfo Workaround: initialized with override: 0 real: 0xf49099c0
XRRGetCrtcInfo Workaround: initialized with override: 0 real: 0xf49081f0
CAppInfoCacheReadFromDiskThread took 0 milliseconds to initialize
Steam Runtime Launch Service: starting steam-runtime-launcher-service
Steam Runtime Launch Service: steam-runtime-launcher-service is running pid 2425
bus_name=com.steampowered.PressureVessel.LaunchAlongsideSteam
[2024-06-22 13:34:06] Background update loop checking for update. . .
[2024-06-22 13:34:06] Checking for available updates...
[2024-06-22 13:34:06] Downloading manifest: https://cdn.steamstatic.com/client/steam_client_ubuntu12?t=1029256832
[2024-06-22 13:34:06] Manifest download: send request
[2024-06-22 13:34:06] Manifest download: waiting for download to finish
[2024-06-22 13:34:07] Manifest download: finished
[2024-06-22 13:34:07] Download skipped: /client/steam_client_ubuntu12?t=1029256832 version 1718904662, installed version 1718904662, existing pending version 0
[2024-06-22 13:34:07] Nothing to do
`

So in other words none of the solutions here have worked for me. Now what?

**Comment by LiamTheNinja5** [comment]
> Same thing: Steam is completely invisible! Tried: 
> 
> rm -rfv ~/.steam ~/.local/share/Steam ~/.steampath ~/.steampid
> 
> and also tried: steam -cef-disable-gpu
> 
> Also removing and reinstalling did NOTHING. On Ubuntu Steam has always worked no matter if a deb or a snap package. Now that i thought to give Mint a shot for a long time on one machine to just enjoy gaming it's nothing but trouble and tinkering? Hard to believe that Mint is based on Ubuntu as same things often are just messed up on Mint. I wonder why is this. 
> 
> Description:	Linux Mint 21.3 XFCE
> Release:	21.3
> Codename:	virginia
> 
> For the record when the command steam -cef-disable-gpu is issued all kind of stuff appears happening in the terminal for the time being and then everything stops and Steam does no even appear to taskbar. Nevermind opening as visible.
> 
> `steam -cef-disable-gpu
> steam.sh[2205]: Running Steam on linuxmint 21.3 64-bit
> steam.sh[2205]: STEAM_RUNTIME is enabled automatically
> setup.sh[2275]: Steam runtime environment up-to-date!
> steam.sh[2205]: Steam client's requirements are satisfied
> [2024-06-22 13:32:04] Startup - updater built Jun 20 2024 17:13:49
> [2024-06-22 13:32:04] Startup - Steam Client launched with: '/home/tux/.steam/debian-installation/ubuntu12_32/steam' '-srt-logger-opened' '-nominidumps' '-nobreakpad' '-cef-disable-gpu'
> 06/22 13:32:04 minidumps folder is set to /tmp/dumps
> 06/22 13:32:04 Init: Installing breakpad exception handler for appid(steam)/version(1718904662)/tid(2344)
> [2024-06-22 13:32:05] Loading cached metrics from disk (/home/tux/.steam/debian-installation/package/steam_client_metrics.bin)
> [2024-06-22 13:32:05] Failed to load cached hosts file (File 'update_hosts_cached.vdf' not found), using defaults
> [2024-06-22 13:32:05] Using the following download hosts for Public, Realm steamglobal
> [2024-06-22 13:32:05] 1. https://cdn.steamstatic.com, /client/, Realm 'steamglobal', weight was 1, source = 'baked in'
> [2024-06-22 13:32:05] Verifying installation...
> [2024-06-22 13:32:05] Verification complete
> UpdateUI: skip show logo
> Steam logging initialized: directory: /home/tux/.steam/debian-installation/logs
> 
> XRRGetOutputInfo Workaround: initialized with override: 0 real: 0xf49099c0
> XRRGetCrtcInfo Workaround: initialized with override: 0 real: 0xf49081f0
> CAppInfoCacheReadFromDiskThread took 0 milliseconds to initialize
> Steam Runtime Launch Service: starting steam-runtime-launcher-service
> Steam Runtime Launch Service: steam-runtime-launcher-service is running pid 2425
> bus_name=com.steampowered.PressureVessel.LaunchAlongsideSteam
> [2024-06-22 13:34:06] Background update loop checking for update. . .
> [2024-06-22 13:34:06] Checking for available updates...
> [2024-06-22 13:34:06] Downloading manifest: https://cdn.steamstatic.com/client/steam_client_ubuntu12?t=1029256832
> [2024-06-22 13:34:06] Manifest download: send request
> [2024-06-22 13:34:06] Manifest download: waiting for download to finish
> [2024-06-22 13:34:07] Manifest download: finished
> [2024-06-22 13:34:07] Download skipped: /client/steam_client_ubuntu12?t=1029256832 version 1718904662, installed version 1718904662, existing pending version 0
> [2024-06-22 13:34:07] Nothing to do
> `
> 
> So in other words none of the solutions here have worked for me. Now what? 
> 
> Edit:  VGA compatible controller: NVIDIA Corporation GF100GL [Quadro 4000] (rev a3)
> NVIDIA driver version: 390.157

@vrement what are your computer specs? When I first installed mint and had this issue The problem was I needed to switch over to the EDGE iso then turn off GPU acceleration because my hardware was so new it needed the newer kernel.

**Comment by kisak-valve** [comment]
Hello @vrement, since the NVIDIA 390 series video driver is involved on your system, you're most likely seeing https://github.com/ValveSoftware/steam-runtime/issues/420 / #10980 instead of the issue reported here.

**Comment by vrement** [comment]
Looks like the proprietary NVIDIA driver was the issue! When i switched to non-official but free driver Nouveau; lo and behold Steam started on Mint via installed deb and Snap Steam packages!

**Comment by vrement** [comment]
> > Replying to [#10532 (comment)](https://github.com/ValveSoftware/steam-for-linux/issues/10532#issuecomment-2183977914)
> 
> @vrement what are your computer specs? When I first installed mint and had this issue The problem was I needed to switch over to the EDGE iso then turn off GPU acceleration because my hardware was so new it needed the newer kernel.

-Computer-
Processor		: Intel(R) Core(TM) i7-2600K CPU @ 3.40GHz
Memory		: 16348MB (4215MB used)
Machine Type		: Desktop
Operating System		: Linux Mint 21.3
-Display-
Resolution		: 3600x1200 pixels
OpenGL Renderer		: NVC0
X11 Vendor		: The X.Org Foundation
-Audio Devices-
Audio Adapter		: HDA-Intel - HDA Intel PCH
Audio Adapter		: HDA-Intel - HDA NVidia

 Logitech Logitech USB Keyboard
 Logitech Logitech USB Keyboard Consumer Control
 Logitech Logitech USB Keyboard System Control
 Eee PC WMI hotkeys
 HDA NVidia HDMI/DP,pcm:3
 HDA NVidia HDMI/DP,pcm:7
 HDA NVidia HDMI/DP,pcm:8
 HDA NVidia HDMI/DP,pcm:9

**Comment by CaptainDumb** [comment]
i had mint running on AMD graphics, so i dont think its a nvidia issue

**Comment by 8JO8** [comment]
I fixed it by going in the Nvidia Driver Settings and disabling FXAA in the antialiasing section

**Comment by radostinslaavov-cloud** [comment]
hi, for me steam just refuses to open, i am on Linux Mint 22.2 Cinnamon

**Comment by vrement** [comment]
> hi, for me steam just refuses to open, i am on Linux Mint 22.2 Cinnamon

Maybe try Steam's Snap package? It works very well now and has all needed (and plus) coming within the package.
